src/components/CharacterAI.hpp

```diff
@@ -281,15 +288,22 @@ namespace REGoth
     // Whether Physics is being processed for this character
     bool mIsPhysicsActive = true;
 
-    // Whether this character is standing on solid ground, like the world mesh or a static vob
+    // Whether this character is standing on solid ground, like the world mesh or a static vob.
+    // Dynamic objects such as movers or other characters do not count as solid.
     bool isStandingOnSolidGround = false;
 
+    // Whether this character is currently in air, not standing on anything.
+    bool isInAir = false;
```

Comment:
Rename to `mIsInAir`?

Reply:
or `mIsAirborne`, see the other suggestion